modules/mobile/docs/openapi.yaml

```diff
@@ -1284,6 +1483,31 @@ components:
         financialInstitutionRoutingNumber:
           type: string
           example: "021000021"
+    PushRegistration:
```

Comment:
This core body looks good, because we use JSON:API spec it gets wrapped in a resource e.g.
```json
{
  "data": {
    "type": "preferences",
    "id": "550e8400-e29b-41d4-a716-446655440000",
    "attributes": {

    }
  }
}
```

If it's a list then it's an [array of resources](https://jsonapi.org/examples/). You could debate if preferences is one resource or a list of resources (I lean toward it being one resource).

Reply:
Cool I will wrap these object definitions